docs/troubleshooting.rst

```diff
@@ -233,6 +233,30 @@ passed before the ``mocker`` fixture to ensure this:
       # works correctly
       mocker.patch("builtins.open", mocker.mock_open(read_data="content"))
 
+Pathlib.Path objects created outside of tests
+---------------------------------------------
+An pattern which is more often seen with the increased usage of `pathlib` is the
+creation of global `pathlib.Path` objects (instead of string paths) that are used
+in the tests. As these objects are created in the real filesystem, they are not of the same
```

Comment:
```suggestion
creation of global `pathlib.Path` objects (instead of string paths) that are imported
into the tests. As these objects are created in the real filesystem, they are not of the same
```